airflow/providers/amazon/aws/transfers/dynamodb_to_s3.py

```diff
@@ -105,7 +105,6 @@ class DynamoDBToS3Operator(AwsToAwsBaseOperator):
         "file_size",
         "dynamodb_scan_kwargs",
         "s3_key_prefix",
-        "process_func",
```

Comment:
can this be safely removed? Providers can be installed with older versions of Airflow.

Reply:
it's a callable , so it's impossible to template

Reply:
so lets remove it in a separated PR? best not to mix providers with core changes in the same PR unless we have to